packages/volto/src/helpers/Blocks/Blocks.test.js

```diff
@@ -968,6 +969,22 @@ describe('Blocks', () => {
   });
 
   describe('buildStyleClassNamesFromData', () => {
+    beforeEach(() => {
+      function blockThemesEnhancer(data) {
+        const blockStyleDefinitions =
+          // We look up for the blockThemes in the block's data, then in the global config
+          // We keep data.colors for BBB, but data.themes should be used
+          data.themes || data.colors || config.blocks.blocksThemes || [];
```

Comment:
I haven't checked if it matters for the tests, but this is the old wrong code, getting `themes` and `colors` from the block data rather than from the blocksConfig

Reply:
Fixed